src/plugins/vis_type_timeseries/public/application/components/aggs/percentile.js

```diff
@@ -36,7 +36,7 @@ import { FormattedMessage } from '@kbn/i18n/react';
 import { KBN_FIELD_TYPES } from '../../../../../../plugins/data/public';
 import { Percentiles, newPercentile } from './percentile_ui';
 
-const RESTRICT_FIELDS = [KBN_FIELD_TYPES.NUMBER];
+const RESTRICT_FIELDS = [KBN_FIELD_TYPES.NUMBER, KBN_FIELD_TYPES.HISTOGRAM];
```

Comment:
I think `KBN_FIELD_TYPES.HISTOGRAM `should be also added for `src/plugins/vis_type_timeseries/public/application/components/aggs/filter_ratio.js`

Reply:
Filter ratio doesn't have a field dropdown.

![image](https://user-images.githubusercontent.com/41702/85579670-eca0e500-b5ef-11ea-86b3-e26c99de6b07.png)

Reply:
You should change `Metric Aggregation` ftom `Count` to `Average` for example

Reply:
Oops... I forgot about that feature. Adding it now.

Reply:
@jasonrhodes should be fixed before merge